test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -21,12 +21,20 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import kbasesearchengine.search.AccessFilter;
```

Comment:
is there a reason these were moved way up here?

Reply:
intellij auto format

Reply:
Can you move them back where they belong please?

Reply:
sure but it will still show changed b/c I have to manually move them